OrbitClientServices/ProcessManager.cpp

```diff
@@ -206,10 +206,15 @@ ErrorMessageOr<std::string> ProcessManagerImpl::LoadProcessMemory(
 }
 
 ErrorMessageOr<std::string> ProcessManagerImpl::LoadNullTerminatedString(
-    int32_t pid, uint64_t address, uint64_t max_size) {
+    int32_t pid, uint64_t address) {
+  constexpr uint64_t max_size = 256;
   auto error_or_string = LoadProcessMemory(pid, address, max_size);
   if (error_or_string.has_value()) {
-    std::string str = error_or_string.value();
+    const std::string& str = error_or_string.value();
+    if (str.find('\0') == std::string::npos) {
+      ERROR("Remote string is not null terminated: %s", str.c_str());
```

Comment:
maybe return a error in this case, is there a valid case for processing non-terminated string?